src/main/java/com/example/demo/controller/BookController.java

```diff
@@ -32,4 +37,20 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.createBook(requestDto);
     }
+
+    @GetMapping("/by-title")
+    public List<BookDto> findAllByTitle(@RequestParam String title) {
+        return bookService.findAllByTitle(title);
+    }
```

Comment:
remove this end point
![image](https://github.com/artemsa223/book-shop/assets/20326858/9f3e0b55-b0aa-4f82-9fda-3f8cf655c741)